indra/cmake/Tracy.cmake

```
@@ -4,14 +4,26 @@ include(Prebuilt)
 include_guard()
 add_library( ll::tracy INTERFACE IMPORTED )
 
-set(USE_TRACY OFF CACHE BOOL "Use Tracy profiler.")
+option(USE_TRACY "Use Tracy profiler." OFF)
 
 if (USE_TRACY)
+  option(USE_TRACY_ON_DEMAND "Use Tracy profiler." ON)
+  option(USE_TRACY_LOCAL_ONLY "Use Tracy profiler." OFF)
+
   use_system_binary(tracy)
   use_prebuilt_binary(tracy)
 
   target_include_directories( ll::tracy SYSTEM INTERFACE ${LIBS_PREBUILT_DIR}/include/tracy)
-  target_link_libraries( ll::tracy INTERFACE TracyClient )
+  
+  target_compile_definitions(ll::tracy INTERFACE -DTRACY_ENABLE=1 -DTRACY_ONLY_IPV4=1)
```

```suggestion

  target_compile_definitions(ll::tracy INTERFACE -DTRACY_ENABLE=1 -DTRACY_ONLY_IPV4=1)
```
To stop pre-commit from complaining.